components/common-go/pprof/pprof.go

```diff
@@ -31,6 +27,15 @@ func Serve(addr string) {
 	}
 }
 
+// Configure congigures pprof endpoints on the given mux
+func Configure(mux *http.ServeMux) {
+	mux.HandleFunc("/debug/pprof/", index)
+	mux.HandleFunc("/debug/pprof/cmdline", pprof.Cmdline)
+	mux.HandleFunc("/debug/pprof/profile", pprof.Profile)
+	mux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
+	mux.HandleFunc("/debug/pprof/trace", pprof.Trace)
+}
+
```

Comment:
I'd rather this be a `http.Handler` akin to https://golang.org/src/net/http/pprof/pprof.go?s=7012:7050#L215


```suggestion
// Handler produces the pprof endpoint handler
func Handler() http.Handler {
        mux :=  http.NewServeMux()
	mux.HandleFunc("/debug/pprof/", index)
	mux.HandleFunc("/debug/pprof/cmdline", pprof.Cmdline)
	mux.HandleFunc("/debug/pprof/profile", pprof.Profile)
	mux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
	mux.HandleFunc("/debug/pprof/trace", pprof.Trace)
	return mux
}

```